.travis.yml

```diff
@@ -77,7 +77,7 @@ env:
     - PATH="$PATH:$TRAVIS_BUILD_DIR/node_modules/.bin"
     - BEARS_ZIP_URL=https://codeload.github.com/coala/coala-bears/zip
     - GRAVIS="https://raw.githubusercontent.com/DanySK/Gravis-CI/master/"
-    - JDK="adopt@1.8.0-212"
+    - JDK="graalvm-ce-java8@21.0.0"
```

Comment:
Can you link from where you took this from?

Reply:
language-check package requires JAVA8([link](https://stackoverflow.com/questions/45725399/pip-install-language-check-not-working)). Among the list of JDK [supported ](https://travis-ci.com/github/anshalshukla/coala/jobs/492086200) I choose graalvm-ce-java8 as it is JAVA8 based and [open-source](https://www.graalvm.org/community/opensource/).

Reply:
We can keep using adoptopenjdk. Don't want to switch to new one without any reason.